db/seeds.rb

```diff
@@ -5,3 +5,13 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+5.times do
+  Product.create([{
+    breed:        Faker::Cat.breed,
+    dob:          Faker::Date.backward(365),
+    description:  Faker::Lorem.paragraph,
+    price:        Faker::Commerce.price,
+    quantity:     Faker::Number.between(1,5)
```

Comment:
Great job on not only figuring out Faker, but also managing to get it to fake dates and prices!